log/rotating_file_handler.ts

```diff
@@ -67,11 +67,13 @@ export class RotatingFileHandler extends FileHandler {
   override setup() {
     if (this.#maxBytes < 1) {
       this.destroy();
-      throw new Error("maxBytes cannot be less than 1");
+      throw new Error(`"maxBytes" must be >= 1: received ${this.#maxBytes}`);
     }
     if (this.#maxBackupCount < 1) {
       this.destroy();
-      throw new Error("maxBackupCount cannot be less than 1");
+      throw new Error(
+        `"maxBackupCount" must be >= 1: received ${this.#maxBackupCount}`,
+      );
```

Comment:
Would it be better if these checks were moved to the constructor and we could fast fail then? Or is there a reason someone would construct the object and not want to call `setup`, and thus want this invalid state?